### PR TITLE
Search, User Guide 7.2 SP1 | Synonym Sets | LRDOCS-5909

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-5909

### CODE REVIEW

**Comment by lipusz** [comment]
Make sure we include https://liferay.slack.com/archives/CMU6DRAQ2/p1568125783013700

File: en/user/articles-dxp/120-search/12-synonyms.markdown [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+header-id: search-tuning-synonym-sets
+---
+
+# Search Tuning: Synonym Sets
+
+[TOC levels=1-4]
+
+| **Known Issues:** There are several [known
```
Comment (lipusz) [inline en/user/articles-dxp/120-search/12-synonyms.markdown]: Can we make it a header like

## Limitations and Known Issues

?

**Comment by lipusz** [comment]
**Started reviewing...** :mag::nerd_face:

File: en/user/articles-dxp/120-search/12-synonyms.markdown (hunk 1 above)
Comment (lipusz) [reply en/user/articles-dxp/120-search/12-synonyms.markdown]: Yes.

**Comment by lipusz** [comment]
Add: link to Elastic's Synonyms guide and refer to in the text to instruct users consult the Elastic guide to better understand synonyms.

**Comment by lipusz** [comment]
https://www.elastic.co/guide/en/elasticsearch/guide/current/synonyms-analysis-chain.html and https://www.elastic.co/guide/en/elasticsearch/guide/current/multi-word-synonyms.html are extremely important when using the feature.

**Comment by lipusz** [comment]
:warning:  https://www.elastic.co/guide/en/elasticsearch/guide/current/synonym-formats.html : The `=>` is not supported, but will check.

**Comment by lipusz** [comment]
:bulb: I came up with a way to backup & preserve Synonym Sets during reindex, see https://issues.liferay.com/browse/LPS-100272 for the details.

**Comment by rbohl** [comment]
> bulb I came up with a way to backup & preserve Synonym Sets during reindex, see https://issues.liferay.com/browse/LPS-100272 for the details.

:1st_place_medal: Nice job finding a workaround.

So we can document this, and then remove when the bug is fixed on FP-3. However, since this is DXP-only documentation, and AFAIK there are no DXP 7.2 customers, nobody will run into this bug. If you feel it's still necessary, we can definitely document it though.

**Comment by lipusz** [comment]
Customers need to run reindex for various reasons so they can remove their synonyms accidentally.

It's not important because of existing 7.2 GA1 or FP1 customers, but for general reasons 

Plus, we actually do have some customers on 7.2 ;-)

**Comment by rbohl** [comment]
Okay,well if we have customers it makes sense. It just means we have to document the workaround, and then remove it when the bug is fixed. I'll need a ticket to go through the known issues after FP-3 and remove any that are fixed. BRB.

:grimacing:  https://issues.liferay.com/browse/LRDOCS-7351

**Comment by lipusz** [comment]
Hey Russell,

I think you may have misunderstood the case: it's not really about existing deployments, but about future deployments running on SP1 and using synonyms: unless we fix this reindex bug, it's crucial to document how admins can backup & preserve their synonym set configs.

**Comment by lipusz** [comment]
Edited, resent as https://github.com/rbohl/liferay-docs/pull/153

**Pull request closed without merging**